go.mod

```diff
@@ -1,15 +1,15 @@
 module github.com/tetratelabs/car
 
 // This project uses go:embed, so requires minimally go 1.16
-go 1.17
+go 1.18
```

Comment:
FYI: this is read by golangci-lint to know whether or not to disable <1.18 linters